src/templates/amfcfg.conf.j2

```diff
@@ -0,0 +1,164 @@
+configuration:
+  amfDBName: {{ amf_database_name }}
+  amfName: AMF
+  debugProfilePort: 5001
+  enableDBStore: false
+  enableSctpLb: false
+  kafkaInfo:
+    brokerPort: 9092
+    brokerUri: sd-core-kafka-headless
+    topicName: sdcore-data-source-amf
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ database_url }}
+  networkFeatureSupport5GS:
+    emc: 0
+    emcN3: 0
+    emf: 0
+    enable: true
+    imsVoPS: 0
+    iwkN26: 0
+    mcsi: 0
+    mpsi: 0
+  networkName:
+    full: Aether5G
+    short: Aether
+  nfKafka:
+    enable: false
+    topic: sdcore-nf-data-source
+    urls:
+    - sd-core-kafka-headless:9092
+  ngapIpList:
+  - 0.0.0.0
+  ngappPort: 38412
+  non3gppDeregistrationTimerValue: 3240
+  nrfUri: {{ nrf_url }}
+  plmnSupportList:
+  - plmnId:
+      mcc: "208"
+      mnc: "93"
+    snssaiList:
+    - sd: "010203"
+      sst: 1
+  sbi:
+    bindingIPv4: 0.0.0.0
+    port: 29518
+    registerIPv4: {{ amf_url }}
+    scheme: http
+  sctpGrpcPort: 9000
+  security:
+    cipheringOrder:
+    - NEA0
+    integrityOrder:
+    - NIA1
+    - NIA2
+  servedGuamiList:
```

Comment:
Again, ideally we would simply remove this

Reply:
✅